### DISCUSSION TITLE
is there option to use gemini pro 1.0 from vertex AI on Google Cloud

### DISCUSSION THREAD

**Discussion by analyticsrepo01** [huggingface/chat-ui#927] [General]
is there option to use gemini pro 1.0 from vertex, please add config if possible for Google cloud

**Comment by nsarrazin** [comment]
There's an [open PR](https://github.com/huggingface/chat-ui/pull/950) for it, i'll let you know once it's merged 😄

**Comment by nsarrazin** [comment]
Support has been added! @analyticsrepo01 Let me know if that resolves your issue

**Comment by asinchronous** [comment]
Hi! Is it possible to support the gemini api (non google cloud/vertex) as well? (https://ai.google.dev/)
